dragonflybsd/README.md

```diff
@@ -0,0 +1,137 @@
+# DragonflyBSD fwupd port notes
+
+## Preparation
+
+- Installed packages
+    - qemu-system-x86-64
+    - ovmf
+- Downloaded
+[latest version](https://mirror-master.dragonflybsd.org/iso-images/?C=M;O=D) of
+DragonflyBSD release in uncompressed `.iso` format
+- Created Virtual disk with `qemu-img` command:
+
```

Comment:
I saw that you use two characters to show that a command is ran in the host terminal (`$`) and QEMU (`#`), I think we should add a piece of information about that

Reply:
Before most commands there's a piece of information about where to run the script, resolved